iron-label.html

```diff
@@ -0,0 +1,124 @@
+<!--
+    @license
+    Copyright (c) 2014 The Polymer Project Authors. All rights reserved.
+    This code may only be used under the BSD style license found at http://polymer.github.io/LICENSE.txt
+    The complete set of authors may be found at http://polymer.github.io/AUTHORS.txt
+    The complete set of contributors may be found at http://polymer.github.io/CONTRIBUTORS.txt
+    Code distributed by Google as part of the polymer project is also
+    subject to an additional IP rights grant found at http://polymer.github.io/PATENTS.txt
+--><!--
+`<iron-label>` provides a version of the `<label>` element that works with Custom Elements as well as native elements.
+
+All text in the `iron-label` will be applied to the target element as a screen-reader accessible description.
+
+There are two ways to use `iron-label` to target an element:
+
+1. place an element inside iron-label with the `for` attribute:
+
+        <iron-label>
+          Context for the Button
+          <paper-button for>button</paper-button>
+        </iron-label>
+
+2. Set the `for` attribute on the `iron-label` element to point to a target element in the same scope with a query
+string:
+
+        <iron-label for=".foo">
+          Context for the button witht the "foo" class"
+        </iron-label>
+        <paper-button class="foo">Far away button</paper-button>
+
+All taps on the `iron-label` will be forwarded to the "target" element.
+
+@group Iron Elements
+@element iron-label
+@demo demo/index.html
+@homepage polymer.github.io
+-->
+<link rel="import" href="../polymer/polymer.html">
+
+<dom-module id="iron-label">
+  <!-- Native <label> has cursor: default -->
+  <style>
+    :host {
+      cursor: default;
+    }
+  </style>
+
+  <template><content></content></template>
```

Comment:
While we're at it, Polymer 1.1. says the `<style>` should go in the `<template>`